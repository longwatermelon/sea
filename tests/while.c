int main() {
    int x=0;
    int ans=0;
    while (x!=5) {
        ans=ans+2*x;
        x=x+1;
    }
    return ans;
}
