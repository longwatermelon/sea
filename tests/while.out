RETURN_CODE:20
STDOUT:
